conda/recipes/rapids-dask-dependency/recipe.yaml

```diff
@@ -28,6 +28,9 @@ requirements:
     - dask ==2025.5.0
     - dask-core ==2025.5.0
     - distributed ==2025.5.0
+    # conda-forge has a bad build of importlib-metadata 8.7. Pinning to earlier for now
+    # to unblock CI, will revert once upstream is fixed.
+    - importlib-metadata <8.7
```

Comment:
IIUC this only affects python 3.10. Are we able to specify that here?

Reply:
I was wondering how we got this in the first place. Good to know it's via dask-core.